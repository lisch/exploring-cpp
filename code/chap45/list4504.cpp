#include "list4504.cc"
