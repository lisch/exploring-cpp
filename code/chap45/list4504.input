1 2 13
15 5 30
