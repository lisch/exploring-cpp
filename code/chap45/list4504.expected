{ 1 2 13 15 5 30 }
Value 13 in range [10,20].
